styles/alphav3-1-eibc.json

```diff
@@ -0,0 +1,265 @@
+          "color": "#2378C7",
+          "name": "Link Hover Color"
+        },
+        {
+          "slug": "quarternary",
+          "color": "#2E5DD7",
+          "name": "Public Sector – Quarternary"
+        },
+        {
+          "slug": "quinary",
+          "color": "#F9C462",
+          "name": "BC Yellow – Quinary"
+        },
+        {
+          "slug": "senary",
+          "color": "#CE3E39",
+          "name": "Danger – Senary"
+        },
+        {
+          "slug": "septenary",
+          "color": "#42814A",
+          "name": "Success – Septenary"
+        },
+        {
+          "slug": "octonary",
+          "color": "#F8BB47",
+          "name": "Warning – Octonary"
+        },
+        {
+          "slug": "nonary",
+          "color": "#2D2D2D",
+          "name": "Icons primary – Nonary"
+        },
+        {
+          "slug": "denary",
+          "color": "#474543",
+          "name": "Icons secondary – Denary"
+        },
+        {
+          "slug": "ondenary",
+          "color": "#053662",
+          "name": "Icons info – Ondenary"
+        },
+        {
+          "slug": "duodenary",
+          "color": "#9F9D9C",
+          "name": "Icons disabled – Duodenary"
+        },
+        {
+          "slug": "tridenary",
+          "color": "rgba(0,0,0,0.45)",
+          "name": "Dark overlay – Tridenary"
+        },
+        {
+          "slug": "quadenary",
+          "color": "#ECEAE8",
+          "name": "Inverted text (non-white) – Quadenary"
+        },
+        {
+          "slug": "gray-80",
+          "color": "#605E5C",
+          "name": "Gray 80"
+        },
+        {
+          "slug": "gray-30",
+          "color": "#ECEAE8",
+          "name": "Gray 30"
+        },
+        {
+          "slug": "gray-10",
+          "color": "#FAF9F8",
+          "name": "Gray 10"
+        },
+        {
+          "slug": "heading-line",
+          "color": "#fcba19",
+          "name": "Heading Line"
+        },
+        {
+          "slug": "foreground",
+          "color": "#2D2D2D",
+          "name": "Foreground"
+        },
+        {
+          "slug": "background",
+          "color": "#ffffff",
+          "name": "Background"
+        },
+        {
+          "slug": "transparent",
+          "color": "transparent",
+          "name": "Transparent"
+        }
+      ]
+    },
+    "spacing": {
+      "blockGap": null,
+      "margin": true
+    },
+    "typography": {
+      "fluid": false,
+      "fontSizes": [
+				{
+					"slug": "small",
+					"size": "1.25rem",
+					"name": "Small"
+				},
+				{
+					"slug": "medium",
+					"size": "1.5rem",
+					"name": "Medium"
+				},
+				{
+					"slug": "large",
+					"size": "1.8rem",
+					"name": "Large"
+				},
+				{
+					"slug": "extra-large",
+					"size": "2.16rem",
+					"name": "Extra Large"
+				},
+				{
+					"slug": "extra-extra-large",
+					"size": "2.592rem",
+					"name": "Extra Extra large"
+				}
+      ],
+      "fontFamilies": []
```

Comment:
shoud bc sans be in here, or is it set as default elswhere, and automatically appplied?

Reply:
In general it should, yes. The EIBC plugin sets it as the default font already though because it was created before we were using any of these alpha styles so it doesn't need to be set here. Adding this style json is definitely a hack that is kind of a compromise between the old Block Theme styles and the Alpha styles just so we can get it displaying reasonably on EIBC.